samples/sampler/stories/default/SmallIconsItem.js

```diff
@@ -0,0 +1,40 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
+import UiItem, {ItemBase as UiItemBase} from '@enact/ui/Item';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Icon, {icons} from '@enact/agate/Icon';
+import Item, {ItemBase} from '@enact/agate/Item';
+
+const Config = mergeComponentMetadata('Item', UiItemBase, UiItem, ItemBase, Item);
+Item.displayName = 'Item';
+
+const iconList = [null, ...Object.keys(icons)];
+
+storiesOf('Agate', module)
+	.add(
+		'SmallIconsItem',
+		() => {
+			const slotAfterIcon1 = select('slotAfter Icon 1', iconList, Config, 'bluetooth');
+			const slotAfterIcon2 = select('slotAfter Icon 2', iconList, Config, 'music');
+			return (
+				<Item
+					disabled={boolean('disabled', Config)}
+					inline={boolean('inline', Config)}
+					label={text('label', Config)}
+					labelPosition={select('labelPosition', ['above', 'after', 'before', 'below'], Config, 'below')}
+					selected={boolean('selected', Config)}
+				>
+					{text('children', Config, 'Hello Item')}
+					<slotAfter>
+						<Icon size="small">{slotAfterIcon1}</Icon>
+						<Icon size="small">{slotAfterIcon2}</Icon>
+					</slotAfter>
+				</Item>
+			);
+		},
+		{
+			text: 'Basic usage of SmallIconsItem'
+		}
+	);
```

Comment:
There is no `SmallIconsItem` public component that we provide. So we won't add `SmallIconsItem` sampler. The name sampler should be the name which is the same with one of our public components.

If not, app developers could confuse whether there is the `SmallIconsItem` public component or not. In addition, we could confuse where is the sampler to verify the Item with small icons.

@0x64 
I would like to unify the `Item` sampler, the `SmallIconsItem` sampler, and the `IconSubTextItem` sampler into the `Item` sampler. Please let me know your thoughts with the following code.
```
		() => {
			const slotBeforeIcon = select('slotBefore Icon', iconList, Config);
			const slotAfter = [
				select('slotAfter Icon 1', iconList, Config),
				select('slotAfter Icon 2', iconList, Config),
				text('slotAfter Text', Config, 'Sub Text')
			].filter(text=>!!text).map(text=>{(iconList.indexOf(text) > 0 ? <Icon key={text}>{text}</Icon> : text));

			return (
				<Item
					disabled={boolean('disabled', Config)}
					inline={boolean('inline', Config)}
					label={text('label', Config)}
					labelPosition={select('labelPosition', ['above', 'after', 'before', 'below'], Config, 'below')}
					selected={boolean('selected', Config)}
				>
					{slotBeforeIcon ? <slotBefore>
						<Icon>{slotBeforeIcon}</Icon>
					</slotBefore> : null}
					{text('children', Config, 'Hello Item')}
					{slotAfter.length > 0 ? <slotAfter>
						{slotAfter}
					</slotAfter> : null}
				</Item>
			);
		},
```

Reply:
After discussing with @0x64 

1. Replace SmallIconsItems.js and IconSubTextItem.js with the following code
2. Add the following code at the end of the samples/sampler/stories/default/Item.js.

```
storiesOf('Agate QA.Item', module)
	.add(
		'with icons and a text in slotAfter',
		() => {
			const slotBeforeIcon = select('slotBefore Icon', iconList, Config);
			const slotAfter = [
				select('slotAfter Icon 1', iconList, Config),
				select('slotAfter Icon 2', iconList, Config),
				text('slotAfter Text', Config, 'Sub Text')
			].filter(text=>!!text).map(text=>(iconList.indexOf(text) > 0 ? <Icon key={text}>{text}</Icon> : text));

			return (
				<Item
					disabled={boolean('disabled', Config)}
					inline={boolean('inline', Config)}
					label={text('label', Config)}
					labelPosition={select('labelPosition', ['above', 'after', 'before', 'below'], Config, 'below')}
					selected={boolean('selected', Config)}
				>
					{slotBeforeIcon ? <slotBefore>
						<Icon>{slotBeforeIcon}</Icon>
					</slotBefore> : null}
					{text('children', Config, 'Hello Item')}
					{slotAfter.length > 0 ? <slotAfter>
						{slotAfter}
					</slotAfter> : null}
				</Item>
			);
		}
	);

```

Reply:
Thank you @ybsung for an update from our discussion.

To be clear, here is my thought on this.

They are not components but examples that have not a default behavior of a component but have a given behavior for specific use cases. And, if changing children inside `slotAfter` is not a big issue to be provided as a sampler, I prefer to make children as a combination of icon(s) text element(s) inside `slotAfter` as a usage guide.

If we need to make other cases for testing of specific given requirements for our target domain beyond the basic behavior of the component, it would be better to add a sample into qa sampler, not default sampler.

Reply:
I understand, thank you for your help and clarifications.